config/tsconfig/base.json

```diff
@@ -2,6 +2,7 @@
   "$schema": "https://json.schemastore.org/tsconfig",
   "display": "Default",
   "compilerOptions": {
+    "sourceMap": true,
```

Comment:
i added this because it's supposed to be necessary, although it didn't seem to have any effect on the dev mode sourceMaps. maybe `ts-loader` is implicitly setting it to true or something